.github/workflows/scheduled-build.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Scheduled Build'
+on:
+  schedule:
+    - cron: '11 1 * * SAT'
+
+jobs:
+  scheduledBuild:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        branch: [master, release]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ matrix.branch }}
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: 'Build & Test'
+        env:
+          BRANCH_NAME: ${{ matrix.branch }}
+        run: ./gradlew check -PbrancheName=$BRANCH_NAME --info
```

Comment:
```suggestion
        run: ./gradlew check -PbranchName=$BRANCH_NAME --info
```

Reply:
@vierbergenlars Good catch. Fixed the typo